src/main/java/com/northconcepts/datapipeline/examples/cookbook/UseDataLineage.java

```diff
@@ -29,18 +30,18 @@ protected void writeImpl(Record record) throws Throwable {
             System.out.println(record);
             
             System.out.println("Record Session Properties: ");
-            System.out.println("Record Number: " + record.getSessionProperty(Lineage.recordNumber.getKey()));
-            System.out.println("File Name: " + record.getSessionProperty(Lineage.file.getKey()));
-            System.out.println("Line Number: " + record.getSessionProperty(Lineage.lineNumber.getKey()));
-            System.out.println("Column Number: " + record.getSessionProperty(Lineage.columnNumber.getKey()));
+            System.out.println("Record Number: " + record.getSessionProperty(RecordLineage.RECORD_NUMBER));
+            System.out.println("File Name: " + record.getSessionProperty(RecordLineage.FILE));
+            System.out.println("Line Number: " + record.getSessionProperty(RecordLineage.LINE_NUMBER));
+            System.out.println("Column Number: " + record.getSessionProperty(RecordLineage.RECORD_NUMBER));
```

Comment:
Let's create an instance of the XXXLineage classes to use here instead of accessing the untyped session properties like this.